client/my-sites/controller.js

```diff
@@ -98,12 +98,8 @@ function createNavigation( context ) {
 	);
 }
 
-function removeSidebar( context ) {
-	context.store.dispatch( setSection( { group: 'sites', secondary: false } ) );
-}
-
 function renderEmptySites( context ) {
-	removeSidebar( context );
+	context.store.dispatch( setSection( { group: 'sites' } ) );
```

Comment:
Yeah, I added this code myself in #37816 😉 It's because when rendering the site selector or no-sites UI, we're rendering a UI that is not really related to the target section and shouldn't inherit, e.g., color styles from it. Or the sidebar setting. Some sections define CSS styles for `.is-section-whatever`, with color overrides, and we don't want to use them here.

Reply:
Makes sense 👍